azure-pipelines.yml

```diff
@@ -4,13 +4,17 @@
 # https://docs.microsoft.com/azure/devops/pipelines/languages/javascript
 
 pool:
-  vmImage: 'Ubuntu 16.04'
+  vmImage: 'ubuntu-latest'
```

Comment:
Hmm, my neglect of this library is quite glaring.

Reply:
I know the feeling. I once created a Sass tool when I first joined GitHub, but didn't realize I had incoming PRs on it until four years later. 🙃